assets/js/cardknox.js

```diff
@@ -223,6 +225,81 @@ jQuery(function ($) {
       };
       setIfieldStyle("card-number", card_style);
       setIfieldStyle("cvv", cvv_style);
+
```

Comment:
Can you remove the comments (or just write why you are doing instead of the full documentation)

Reply:
Okay, Removed.